modules/UI/videolayout/RemoteVideo.js

```diff
@@ -134,7 +133,7 @@ RemoteVideo.prototype._initPopupMenu = function (popupMenuElement) {
     this.popover.show = function () {
         // update content by forcing it, to finish even if popover
         // is not visible
-        this.updateRemoteVideoMenu(this.isAudioMuted, true);
+        this.updateRemoteVideoMenu(true);
```

Comment:
The InlineDialog conversion got merged in so popover no longer exists. I think updateRemoteVideoMenu essentially works the same way, though.